src/components/oc-client/src/l.js

```diff
@@ -0,0 +1,152 @@
+            this.load(url[l]);
+          }
+          cb && url.push(cb); // relaunch the dependancie queue
+          return this.load.apply(this, url);
+        }
+        if (url.match(/\.css\b/)) {
+          return this.loadcss(url, cb);
+        }
+        return this.loadjs(url, cb);
+      }
+      , loaded = {}  // will handle already loaded urls
+      , loader = {
+        aliases: {}
+        , loadjs: function (url, cb) {
+          var parts = urlParse(url);
+          url = parts.u;
+          if (loaded[url] === true) { // already loaded exec cb if any
+            cb && cb();
+            return this;
+          } else if (loaded[url] !== undefined) { // already asked for loading we append callback if any else return
+            if (cb) {
+              loaded[url] = (function (ocb, cb) { return function () { ocb && ocb(); cb && cb(); }; })(loaded[url], cb);
+            }
+            return this;
+          }
+          // first time we ask this script
+          loaded[url] = (function (cb) { return function () { loaded[url] = true; cb && cb(); }; })(cb);
+          cb = function () { loaded[url](); };
+          appendElmt('script', {
+            type: 'text/javascript', src: url, id: parts.i, onerror: function (error) {
+              if (parts.f) {
+                var c = error.currentTarget;
+                c.parentNode.removeChild(c);
+                appendElmt('script', { type: 'text/javascript', src: parts.f, id: parts.i }, cb);
+              }
+            }
+          }, cb);
+          return this;
+        }
+        , loadcss: function (url, cb) {
+          var parts = urlParse(url);
+          url = parts.u;
+          loaded[url] || appendElmt('link', { type: 'text/css', rel: 'stylesheet', href: url, id: parts.i });
+          loaded[url] = true;
+          cb && cb();
+          return this;
+        }
+        , load: function () {
+          var argv = arguments, argc = argv[length];
+          if (argc === 1 && isA(argv[0], Function)) {
+            argv[0]();
+            return this;
+          }
+          load.call(this, argv[0], argc <= 1 ? undefined : function () { loader.load.apply(loader, [].slice.call(argv, 1)); });
+          return this;
+        }
+        , addAliases: function (aliases) {
+          for (var i in aliases) {
+            this.aliases[i] = isA(aliases[i]) ? aliases[i].slice(0) : aliases[i];
+          }
+          return this;
+        }
+      }
+      ;
+    if (checkLoaded) {
+      var i, l, links, url;
+      for (i = 0, l = scripts[length]; i < l; i++) {
+        (url = scripts[i].getAttribute('src')) && (loaded[url.replace(/#.*$/, '')] = true);
+      }
+      links = D[getElementsByTagName]('link');
+      for (i = 0, l = links[length]; i < l; i++) {
+        (links[i].rel === 'stylesheet' || links[i].type === 'text/css') && (loaded[links[i].getAttribute('href').replace(/#.*$/, '')] = true);
+      }
+    }
+    //export ljs (as `head`)
+    window.head = loader;
```

Comment:
do you want me to use `ljs` instead of `head` as the name of the loader? sure I'll try to do that. I will also test CSS loading.